gui/src/test/java/kcl/teamIndexZero/traffic/gui/GUI_PrimitivesTest.java

```diff
@@ -0,0 +1,48 @@
+package kcl.teamIndexZero.traffic.gui;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by kumarawijeet on 3/16/2016.
+ */
+public class GUI_PrimitivesTest {
```

Comment:
Point of unit test is to do that not manually (one time) but to add this check into the software itself. Later, in 6 weeks, when someone will be changed your code (who knows?), in case they make it working differently, they will notice the failed tests and fix them. It is like other parts of the code may depend on your logic - exceptions thrown, correct calculations etc - but we never know if the logic is still same - so we write a test to check if the logic is still the same.